### DISCUSSION TITLE
How can I open the io_pdx_mesh tab in the tool shelf? (Blender UI)

### DISCUSSION THREAD

**Discussion by blueblackdot** [ross-g/io_pdx_mesh#67] [Q&A] [answered]
I fail miserably even installing the tool ...

I downloaded Blender 2.91 and the 3.1 but when I try to install it from Prefernces/Add-ons/Install from file nothing happens ... no error message ... no checkbox to tick ... no tool ...

I even signed up for a Maya 2023 test license ... copied everything into scripts ... copied the command from here ...
and get the error: # Error: ModuleNotFoundError: file <maya console> line 1: No module named 'io_pdx_mesh'

I'm totally lost ... Is there any kind soul who can explain me which obvious beginner mistake I make here?
Any help is greatly appreciated ... many thanks!

**Comment by ross-g** [comment, marked as answer]
I will try to help, a couple of questions first,

Did you download the [latest release zip file](https://github.com/ross-g/io_pdx_mesh/releases/latest) or did you download the source code by mistake? (you need to download the zip)

Did you follow these steps in Blender?
https://github.com/ross-g/io_pdx_mesh/wiki/Setup-and-installation#blender-installation
After `Install Add-on from file` you should pick the zip and then it will install and show you the add-on info:
![Enabled add-on](https://raw.githubusercontent.com/wiki/ross-g/io_pdx_mesh/images/blender/setup_03.png)

**Reply by blueblackdot** [reply]
Many thanks! ... not sure why, probably I used the already unzipped folder again and again *sigh*

May I ask one last question (because Blender UI is quite intimitating for a first time user): how can I open the tab in the tool shelf?
I looked and clicked everwhere ... sorry for such annoying questions :/

**Reply by ross-g** [reply]
From the 3D Viewport, the menus in the top left corner:
`View > Sidebar (N)`

**Reply by blueblackdot** [reply]
OMG ... I would have never found that! ... and I couldn't find anything useful in the Blender documentaion about how to activate that ...

Many thanks for your kind help!!!
